Python/bin/packUtil.py

```diff
@@ -231,33 +231,33 @@ def unpackRef(packedref):
 
 
 def packProgID(c): # for program code id -- must be alpha
-   # should be radix-52 with 0-1 first
+   # should be radix-62 with 0-1 first
    try:
       codeIndex = programCodesArray.index(c)
    except:
-      raise RuntimeError("Illegal program code " + c)
-   
-   if codeIndex > 51:
+      raise RuntimeError("Illegal program code '" + c + "'")
+
+   if codeIndex > 61:
```

Comment:
@stevechesley I believe this also have to have the option for first=something else. If you look at your progDecoder.txt, first can be also '2'

Reply:
The function `packProgID` converts a 1-char obs80 program code to a 2-char ADES program code. But there are only 94 possible obs80 program codes (not counting a blank). The last one is `z` = `93` = `1V`. The point is that you cannot get a codeIndex > 93 in this function, and so `0` and `1` are the only possible leading characters.